tensorboard/compat/tensorflow_stub/BUILD

```diff
@@ -59,3 +59,15 @@ py_test(
         "//tensorboard:test",
     ],
 )
+
+py_test(
+    name = "gfile_fsspec_test",
+    size = "small",
+    srcs = ["io/gfile_fsspec_test.py"],
+    srcs_version = "PY3",
+    tags = ["support_notf"],
+    deps = [
+        ":tensorflow_stub",
+        "//tensorboard:test",
```

Comment:
Could you add a line here with `"//tensorboard:expect_fsspec_installed",` similar to the one for numpy earlier in the file?  And then we'll need to create the corresponding target, which you can base off of this one for pandas:

https://github.com/tensorflow/tensorboard/blob/31e48c8616e75be4fffadb1efa720c9446de0460/tensorboard/BUILD#L484-L489

This looks odd on GitHub since it's just a placeholder, but it allows us to flag that we need the `fsspec` dependency in a way that scales for the internal build.

We'll also need the same dep added to the test in tensorboard/summary/writer/BUILD.

Reply:
done